### PR TITLE
Add Jeannie Puentes bio and add NIH course offerings.

### PR DESCRIPTION
**Pull request by AndrewLester**

### CODE REVIEW

**Comment by AndrewLester** [comment]
@ErikBoesen requesting review.

File: language-culture-and-medicine.html [hunk 1]
```diff
@@ -7,8 +7,20 @@ <h1>Language Culture and Medicine</h1>
...
+        <div>Advanced Conversation: Language, Culture and Medicine</div>
+        <div>Schedule: September 10 - December 17, Tuesdays and Thursdays, 4:00 p.m.-5:30 p.m.</div>
+        <div>Tuition: $480</div>
+    </p>
+    <p>
+        <div>English for Workplace Communication</div>
+        <div>Schedule: September 10 - December 17, Tuesdays and Thursdays, 1:00 p.m. - 2:30 p.m.</div>
+        <div>Tuition: $480</div>
+    </p>
+    <div>You can register for Advanced Conversation: Language, Culture and Medicine (English 355) or English for Workplace Communication (Engish 375) online at <a href="https://my.faes.org/Common/CourseSchedule.aspx" target="_blank">https://my.faes.org/Common/CourseSchedule.aspx</a>. Please contact Jennifer Kagan, English Now! Director of Program Development and English Language Instructor, and FAES Faculty, at <a href onclick="emailMe('jennifer.kagan')">jennifer.kagan@english-now.com</a>, with any questions about these courses.</div>
```
Comment (ErikBoesen) [inline language-culture-and-medicine.html]: Why `target=_blank`?

Comment (AndrewLester) [reply language-culture-and-medicine.html]: I just thought it would be useful to open in a new tab because it takes the user to another site.

Comment (ErikBoesen) [reply language-culture-and-medicine.html]: I would remove this because we don't do it consistently throughout the website.

**Pull request merged**